tests/test_app.py

```diff
@@ -192,7 +194,7 @@ def test_find_by_ext_p(self) -> None:
             for f in [".pem", ".crt", ".cer", ".csr", ".deR"]:
                 file_path = os.path.join(tmp_dir, f"dummy{f}")
                 assert not os.path.exists(file_path)
-                open(file_path, "w").write("The quick brown fox jumps over the lazy dog")
+                open(file_path, "wb").write(AZ_DATA)
```

Comment:
Is there any differences between write mode and binary write mode? Please let me know about this..

Reply:
No. There is no difference. The test requires only not-empty files to find them by extension. Possibly, bytes are not required extra conversion.

Reply:
If there is no difference, what about change "wb" option to "w"?
Almost all file write options that in CredSweeper are using "w".

Reply:
Ok. This suite was rollback to use 'w'.

Reply:
> Ok. This suite was rollback to use 'w'.

@babenek 
It seems there is no change regarding 'w' option..
Please check.